PinpointKit/PinpointKit/Sources/Editing/EditImageViewController.swift

```diff
@@ -103,7 +104,7 @@ public final class EditImageViewController: UIViewController, UIGestureRecognize
     private var currentAnnotationView: AnnotationView? {
         didSet {
             if let oldTextAnnotationView = oldValue as? TextAnnotationView {
-                NotificationCenter.default.removeObserver(self, name: .UITextViewTextDidEndEditing, object: oldTextAnnotationView.textView)
+                NotificationCenter.`default`.removeObserver(self, name: .UITextViewTextDidEndEditing, object: oldTextAnnotationView.textView)
```

Comment:
Indeed, and thanks for bringing it up. It appears no longer necessary, but rather an artifact of either the migrator or an auto-fix.